exonum/src/helpers/fabric/details.rs

```diff
@@ -332,6 +332,7 @@ impl GenerateNodeConfig {
         let error_msg = &format!("Expected an ip address in {}: {:?}", PEER_ADDRESS, addr_str);
 
         let external_addr = addr_str.parse::<SocketAddr>().unwrap_or_else(|_| {
+            #[cfg_attr(feature = "cargo-clippy", allow(expect_fun_call))]
```

Comment:
maybe I missed something, but this piece of code looks fishy. why are we pre-formatting this error msg before the closure and disabling the lint?